### PR TITLE
Add note about emulator-5558 offline issue

### PR DESCRIPTION
**Pull request by alex-galey**
Got the issue `emulator-5558 offline` on Oneplus One.
Solved by restarting anbox, thanks to [this upstream issue](https://github.com/anbox/anbox/issues/75).

Added note to the doc

### CODE REVIEW

**Comment by doniks** [comment]
Looking over the issue, it sounds like there are three different ways to resolve this?! In order of applied force:
 * restart adb 
 * kill anbox 
 * reboot

Is my understanding correct? Then I'd suggest describing all three

File: userguide/dailyuse/anbox.rst [hunk 1]
```diff
@@ -65,6 +65,12 @@ You can check that adb server is correctly running locally on your phone by open
     phablet@ubuntu-phablet:~$ adb devices  
     List of devices attached  
     emulator-5558	device  
+    
+If you get ``emulator-5558  offline`` :
```
Comment (doniks) [inline userguide/dailyuse/anbox.rst:69]: Let's rephrase this to make it more clear: If adb fails with the error message blah...

File: userguide/dailyuse/anbox.rst [hunk 2]
```diff
@@ -65,6 +65,12 @@ You can check that adb server is correctly running locally on your phone by open
...
+
+1. Find the pid of anbox owned by phablet with `` top | grep anbox``
```
Comment (doniks) [inline userguide/dailyuse/anbox.rst:71]: The formatting went wrong: https://ci.ubports.com/job/docs.ubports.com/job/PR-296/1/artifact/_build/html/userguide/dailyuse/anbox.html

**Comment by doniks** [comment]
Hey @alex-galey are you still around to finish this?

**Comment by UniversalSuperBox** [comment]
anbox.rst now conflicts and we've had no response from the submitter. I'll close this PR for now, but if you want to fix it up just let me know @alex-galey!
